man/fast-vm.8

```diff
@@ -182,6 +195,15 @@ Number identifying the VM. Currently limited to range 20-220 to allow to be used
 .I /path/to/custom/script
 Local path to script which will be executed as soon as the VM is started and reachable by SSH via its IP address assigned by DHCP.
 
+.TP
+.I /path/to/source/on/this/host
+Local path to a file to be copied to a VM via SCP.
+
+.TP
+.I /path/on/destination/to/put/it
+Optional destination path on the VM to write the file being copied via SCP.
+If not specified, the default is the home directory.
```

Comment:
The default is 'root user home directory' as we specify `root` for scp in the code. It will be better to make it clear here.

Reply:
I've reworked this to not only do host-to-guest; you can now do:

Copy from host to guest:  fast-vm scp 52 afile vm:/tmp
Copy from guest to host:  fast-vm scp 52 vm:/tmp/afile .